UI/MapWnd.cpp

```diff
@@ -283,6 +284,198 @@ namespace {
         GetOptionsDB().Set(option_name, !initially_enabled);
         return !initially_enabled;
     }
+
+    /** BrowseInfoWnd for the fleet icon tooltip */
+    class FleetDetailBrowseWnd : public GG::BrowseInfoWnd {
+    public:
+        FleetDetailBrowseWnd(int empire_id, GG::X width) :
+            GG::BrowseInfoWnd(GG::X0, GG::Y0, width, GG::Y(ClientUI::Pts())),
+            m_empire_id(empire_id),
+            m_margin(5)
+        {
+            GG::X value_col_width{(m_margin * 3) + (ClientUI::Pts() * 3)};
+            m_col_widths = {width - value_col_width, value_col_width};
+
+            SetChildClippingMode(ClipToClient);
+
+            NewLabelValue("MAP_FLEET_SHIP_COUNT", true);
+            NewLabelValue("MAP_FLEET_ARMED_COUNT");
+            NewLabelValue("MAP_FLEET_SLOT_COUNT");
+            NewLabelValue("MAP_FLEET_PART_COUNT");
+            NewLabelValue("MAP_FLEET_UNARMED_COUNT");
+            NewLabelValue("MAP_FLEET_COLONY_COUNT");
+            NewLabelValue("MAP_FLEET_CARRIER_COUNT");
+            NewLabelValue("MAP_FLEET_TROOP_COUNT");
+
+            UpdateLabels();
+            DoLayout();
+        }
```

Comment:
It is a minor issue.  My eyes are looking for a fraction.  Swapping the line order also improves it for me.  Again it is a quibble.